packages/patternlab/styleguide/source/_patterns/02-molecules/secondary-nav.twig

```diff
@@ -0,0 +1,38 @@
+<nav class="ma__secondary-nav">
+  <div class="ma__secondary-nav__container">
+    <ul class="ma__secondary-nav__items js-secondary-nav">
+      {% for item in navItems %}
+        {% set buttonId =  'button' ~ loop.index %}
+        {% set menuId =  'menu' ~ loop.index %}
+        <li class="ma__secondary-nav__item {{ item.active ? 'is-active' : '' }} {{ item.subNav ? 'has-subnav js-secondary-nav-top-item' : 'js-secondary-nav-top-link' }}">
+          {% if item.href %}
+            <a href="{{ item.href }}" class="ma__secondary-nav__link">{{ item.text }}</a>
+          {% else %}
+            <span id="{{ buttonId }}" class="ma__secondary-nav__link">{{ item.text }}</span>
+          {% endif %}
+
+          {% if item.subNav %}
+            <button type="button" id="{{ buttonId }}" class="ma__secondary-nav__top-link-trigger js-secondary-nav-trigger" aria-haspopup="true" aria-expanded="false" aria-hidden="true">
+              <span class="ma__visually-hidden">Show the sub topics of {{item.text}}</span>
+              <svg class="ma__secondary-nav__top-link-icon icon-chevron" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
can you move the icons into packages/patternlab/styleguide/source/_patterns/01-atoms/05-icons and reference them using the icon function {{ icon('chevron') }} and {{ icon('plus') }}?

Reply:
I can help with making this change ^